scripts/release/publish-commands/validate-tags.js

```diff
@@ -21,14 +21,14 @@ const run = async ({cwd, packages, tags}) => {
   if (version.indexOf('0.0.0') === 0) {
     if (tags.includes('latest')) {
       console.log(
-        theme`{error Canary release} {version ${version}} {error cannot be tagged as} {tag latest}`
+        theme`{error Next release} {version ${version}} {error cannot be tagged as} {tag latest}`
       );
       process.exit(1);
     }
   } else {
-    if (tags.includes('canary')) {
+    if (tags.includes('next')) {
       console.log(
-        theme`{error Stable release} {version ${version}} {error cannot be tagged as} {tag canary}`
+        theme`{error Stable release} {version ${version}} {error cannot be tagged as} {tag next}`
```

Comment:
This goes with my other comment about preventing us from tagging a semver range as `next`.